desktop/core/src/desktop/lib/idbroker/conf.py

```diff
@@ -89,7 +82,12 @@ def get_cab_password(fs=None):
 def is_idbroker_enabled(fs=None):
   from desktop.conf import RAZ  # Must be imported dynamically in order to have proper value
 
-  return get_cab_address(fs) is not None and not RAZ.IS_ENABLED.get() # Skipping IDBroker for FS when RAZ is present
+  fs = validate_fs(fs)
```

Comment:
so when you are checking the validation of filesystems here it will only give warning in logs but looks like we need to implement like ?
`idbroker_addr_from_coresite  = get_conf().get(_CNF_CAB_ADDRESS % fs)  if fs else None`

Reply:
For `fs = None` scenario, `validate_fs(fs)` will also return `None`
And then it will try to find the respective config value in this line: `idbroker_addr_from_coresite = get_conf().get(_CNF_CAB_ADDRESS % fs)`

So `get_conf().get('fs.None.ext.cab.address')` will also return None because there is no such config.